wheatley/bot.py

```diff
@@ -32,7 +32,7 @@ class Bot:
     logger_name = "BOT"
 
     def __init__(self, tower: RingingRoomTower, row_generator: RowGenerator, do_up_down_in: bool,
-                 stop_at_rounds: bool, rhythm: Rhythm, user_name: Optional[str]=None,
+                 stop_at_rounds: bool, rhythm: Rhythm, is_calling: bool=True, user_name: Optional[str]=None,
```

Comment:
Should this default to `False`? I would expect it to always be `False` for method ringing and False at least some fo the time for composition (e.g. Spliced)

Reply:
It definitely should default to `false`, but I wanted to test the new behaviour without having to add a CLI arg, so it currently defaults to `true`. I'll definitely change it before merging, though.